docs: update Deploy an Edge Cluster on VMware tutorial (#6446)
spectrocloud/librarium: docs/docs-content/tutorials/edge/deploy-cluster.md

  [Create Registration Token](../../clusters/edge/site-deployment/site-installation/create-registration-token.md) guide.
  Copy the newly created token to a clipboard or notepad file to use later in this tutorial.

  :::warning

>   Ensure that you set a default project for the token. Otherwise, your Edge hosts will not register themselves with

Author: @lennessyy did you mean adding the warning to the Create Registration Token page in this PR or in a separate one?

Reviewer: You can do this in one PR, since this one is getting backported anyway

Author: Done

>   Ensure that you set a default project for the token. Otherwise, your Edge hosts will not register themselves with

Reviewer 2: ```suggestion
  Ensure that you set a default project for the token. Otherwise, your Edge hosts will not register with
```

>   Ensure that you set **Default Project** for the token. Otherwise, your Edge hosts will not register themselves with

Reviewer: ```suggestion
  Ensure that you set a default project for the token. Otherwise, your Edge hosts will not register themselves with
```
I know you are using bold because of the UI form element, but here you are more referring to the concept of default project generally

Reviewer: Can you add this warning to the Create Registration Token page too?

>   Palette automatically.

Reviewer 2: ```suggestion
  Palette.
```

  Should you need to change the VM template name or VM settings defined in the **vsphere.hcl** file, or review the
  Packer script, you must open a bash session into the container using the
  `docker run --interactive --tty --env-file .packerenv --volume "${ISOFILEPATH}:/edge/vmware/packer/build" ghcr.io/spectrocloud/tutorials:1.1.13 bash`
  command, and change to the **edge/vmware/packer/** directory to make the modifications. After you finish the
  modifications, issue the `packer build -force --var-file=vsphere.hcl build.pkr.hcl` command inside the container to

>   trigger the Packer build process. This command will create a VM template, so that you can skip the next step.

Reviewer 2: ```suggestion
  trigger the Packer build process. This command creates a VM template, so that you can skip the next step.
```

<!-- prettier-ignore-end -->

| **Pack Type** | **Registry** | **Pack Name** | **Pack Version**   |
| ------------- | ------------ | ------------- | ------------------ |

> | OS            | Public Repo  | BYOS Edge OS  | Any non-deprecated |

Reviewer 2: ```suggestion
| OS            | Public Repo  | BYOS Edge OS  | Any non-deprecated version. |
```

Author: I replaced it with "Not applicable", since the image is specified inside the pack yaml

Click on the **Next layer** button to add the following Kubernetes layer to your cluster profile.

| **Pack Type** | **Registry** | **Pack Name**         | **Pack Version** |
| ------------- | ------------ | --------------------- | ---------------- |

> | Kubernetes    | Public Repo  | Palette Optimized K3s | `1.27.5`         |

Reviewer: This version does matter. It has to match The arg file that you used to build provider images, so you should keep this value

Author: > This version does matter. It has to match The arg file that you used to build provider images, so you should keep this value

I agree that this version matters, but there is a note after the table: "The pack version must match the version pushed to the to the _ttl.sh_ image registry". In my opinion, providing an exact version in this table might be more confusing. Maybe it's worth placing the note to the table?

Reviewer: Curious why you think using the specifc version would be confusing? Earlier in the guide, we do tell users specifically to push 1.27.5 to the tth registry

```
docker push ttl.sh/ubuntu:k3s-1.27.5-v4.1.2-demo
```

Author: Changed the version 1.27.5 -> 1.27.2 after the internal discussion

Interface (CNI). However, you can choose a different CNI pack that fits your needs, such as Flannel, Cilium, or Custom
CNI.

| **Pack Type** | **Registry** | **Pack Name** | **Pack Version** |
| ------------- | ------------ | ------------- | ---------------- |

> | Network       | Public Repo  | Calico        | `3.25.x`         |

Reviewer: I would just keep a version that we have tried and know is working. I know you may be thinking of maintenance burden, but we don't need to always update this value and we don't know if a newer version may break things. So it's better to stick with versions that we know work.

Author: > I would just keep a version that we have tried and know is working. I know you may be thinking of maintenance burden, but we don't need to always update this value and we don't know if a newer version may break things. So it's better to stick with versions that we know work.

Sure! Should the same be applied to BYOS Edge OS?

Reviewer: The version field is not applicable to the BYOS pack because you specify the image inside the pack yaml itself
